.gitignore

```diff
@@ -48,6 +48,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *.cover
+gql-dev/
```

Comment:
Why this one is needed?

Reply:
This is indeed not directly related to the issue at hand, but was required (to not have a clumsy `git status` output) for me when using the `virtualenv` as recommended in [CONTRIBUTING](https://github.com/graphql-python/gql/blob/master/CONTRIBUTING.md#development-with-virtualenv-recommended):

```sh
virtualenv gql-dev
```

(Create a `gql-dev` directory for the virtuelenv)

Reply:
That shouldn't be ignored as I haven't seen anyone with the same issue and also, the virtualenv name can vary between contributors.

Can you remove this please?

Reply:
Sure I can.

Please note that any following the guidelines and using the same virtualenv command will have the same issue... except if the `CONTRIBUTING` is updated to use the more common `venv` virtuelenv name (which is already in the `.gitignore`).

Reply:
Changing the CONTRIBUTING.md to use `venv` or `.venv` (which seems to be even more common nowadays and is in `.gitignore` as well) is a good idea.